validator.go

```diff
@@ -78,27 +79,34 @@ var (
 type ErrorMap map[string]ErrorArray
 
 // ErrorMap implements the Error interface so we can check error against nil.
-// The returned error is if existent the first error which was added to the map.
+// The returned error is all existing errors with the map.
 func (err ErrorMap) Error() string {
+	var b bytes.Buffer
+
 	for k, errs := range err {
 		if len(errs) > 0 {
-			return fmt.Sprintf("%s: %s", k, errs.Error())
+			b.WriteString(fmt.Sprintf("%s: %s ", k, errs.Error()))
```

Comment:
@BrentBrown Is the extra space following the second `%s` intentional? Because our integration tests are failing potentially due to this change.

Reply:
It wasn't intentional but I believe this pull request is attempting to fix the issue.
https://github.com/go-validator/validator/pull/84

Reply:
Cool thanks